src/index.html

```diff
@@ -11,7 +11,174 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
```

Comment:
multi-line long attributes
indent attributes
keep closing bracket at same indent as the opening one
```suggestion
    <form
      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
      method="post"
    >
```